data/campaigns/The_South_Guard/scenarios/01_Born_to_the_Banner.cfg

```diff
@@ -204,28 +204,28 @@
 
         [message]
             speaker=Moreth
-            message= _ "You’ll never take this village! I’ve seen your kind kill too many townsmen, and I’ll defend them to the death!"
+            message= _ "You’ll never take this village! I'm sick of watching your kind ransack our homes and killing the townsmen! I’ll defend them to the death if I must!"
         [/message]
 
         [message]
             speaker=Deoran
             image=portraits/deoran-mad.png
-            message= _ "My kind? I am a soldier of Wesnoth, a messenger of King Haldric, and the new commander of the South Guard! What do you mean, <i>“my kind”</i>?"
+            message= _ "My kind? I am a knight of Wesnoth and a messenger of the king Haldric! What do you mean, <i>'my kind'</i>? And where is the South Guard?"
```

Comment:
I think these should still be double-quotes, as the programmatic string quotes don't actually have bearing on the presentation of the dialogue– unless this same stylistic change is being made in other campaigns as well. I might be too out of the loop to comment.